topics/proteomics/tutorials/biomarker_selection/tutorial.md

```diff
@@ -0,0 +1,380 @@
+>            - *"Column number on which to apply the filter"*: `c9`
+>            - *"Enter keywords"*: `copy/paste`
+>                - *"Copy/paste keywords to find (keep or discard)"*: `cytoplasm cytosol`
+>    - In *"Filter by numerical value"*:
+>        - {% icon param-repeat %} *"Insert Filter by numerical value"*
+>            - *"Column number on which to apply the filter"*: `c8`
+>            - *"Value"*: `0`
+>    - *"Sort by column ?"*: `Yes`
+>        - *"Sort result files by:"*: `c5`
+>        - *"Sort in descending order ?"*: `Yes`
+>
+>
+>   > ### Output
+>   > - **Filtered Add_information_from_neXtProt**: output list of the proteins having a cytoplasmic
+>   > location and no TMD (48 proteins)
+>   > - **Filtered Add_information_from_neXtProt - discarded lines**: output list of the proteins NOT
+>   > cytoplasmic and having at least 1 TMD.
+>   {: .comment}
+{: .hands_on}
+
+We have now 48 proteins. The next step is to identify those proteins already previously observed in
+LS MS/MS experiments.
+
+
+# Check for previous detection by LC-MS/MS experiments
+
+> ### {% icon hands_on %} Hands-on: Get MS/MS observations in tissue/fluid
+>
+> 1. **Get MS/MS observations in tissue/fluid** {% icon tool %} with the following parameters:
+>    - *"Enter your IDs (UniProt Accession number only)"*: `Input file containing your IDs `
+>        - *"Column of IDs"*: `c6`
+>    - *"Proteomics dataset (biological sample)"*:
+>        - `Human Heart`
+>        - `Human Plasma non glyco`
+>
+>   > ### Output
+>   > **Get MS/MS observations in tissue/fluid**: In this file, 2 columns (11 and 12, at the end)
+>   > were added with the info of number of times peptides were seen by MS/MS.
+>   {: .comment}
+{: .hands_on}
+
+Let's now keep only proteins that have already been seen by MS/MS in the plasma (last column of the file).
+
+
+> ### {% icon hands_on %} Hands-on: Filter for proteins seen in the plasma
+>
+> 1. **Filter by keywords and/or numerical value** {% icon tool %} with the following parameters:
+>    - *"Operation"*: `Discard`
+>    - In *"Filter by keywords"*:
+>        - {% icon param-repeat %} *"Insert Filter by keywords"*
+>            - *"Column number on which to apply the filter"*: `c12`
+>            - *"Search for exact match?"*: `Yes`
+>            - *"Enter keywords"*: `copy/paste`
+>                - *"Copy/paste keywords to find (keep or discard)"*: `NA`
+>    - *"Sort by column ?"*: `Yes`
+>
+>   > ### Output
+>   > - **Filtered Get MS/MS observations in tissue/fluid**: output list of the proteins
+>   > whose some peptides have been seen in plasma (21 proteins)
+>   > - **Filtered Get MS/MS observations in tissue/fluid - discarded lines**:
+>   > output list of proteins with no peptides seen in the plasma
+>   {: .comment}
+{: .hands_on}
+
+
+
+# Conclusion
+{:.no_toc}
+At the end of the process we end up with a list of 21 biomarkers that are **highly enriched in heart muscle**, localized
```

Comment:
@combesf In the tutorial we end up with 21 candidate biomarkers, but the overview image at the start says 22, do you know where the difference comes from?

Reply:
@shiltemann 
Yes, it comes from different versions of the databases we use. Different versions (release date), slightly different results. 
This has really minor or no consequences on our strategy. 
Cheers.

Reply:
ok, cool, it's not a problem, but people will wonder about this, but we could just add a small note about this then.